PureBasicIDE/ColorSchemes.pb

```diff
@@ -0,0 +1,324 @@
+﻿; --------------------------------------------------------------------------------------------
+;  Copyright (c) Fantaisie Software. All rights reserved.
+;  Dual licensed under the GPL and Fantaisie Software licenses.
+;  See LICENSE and LICENSE-FANTAISIE in the project root for license information.
+; --------------------------------------------------------------------------------------------
+
+
+; Store color names in an array, for indexable lookup
+Global Dim ColorName.s(#COLOR_Last_IncludingToolsPanel)
+
+; Color Scheme structure
+Structure ColorSchemeStruct
+  Name.s
+  File.s
+  
+  ColorValue.l[#COLOR_Last_IncludingToolsPanel + 1]
+  
+  IsIDEDefault.i
+  IsAccessibility.i
+EndStructure
+
+; Global list of built-in / found schemes
+Global NewList ColorScheme.ColorSchemeStruct()
+
+; Color values with special meanings
+#ColorSchemeValue_UseSysColor = -1
+#ColorSchemeValue_Undefined   = -2
+
+
+; These (from Preferences.pb) are used
+Global PreferenceToolsPanelFrontColor, PreferenceToolsPanelBackColor
```

Comment:
Variables that are needed in multiple files should be moved to `Common.pb`. In a similar way, procedure declarations used outside of one file go in `Declarations.pb`

Reply:
Will do